### ISSUE TITLE
Admin view of Tokens and Airdrops home - pre-Owner token minting

### ISSUE THREAD

**Issue by benjthayer** [status-im/status-desktop#11472]
A couple of new screens have been added to the Mint Owner token flows to show what an admin sees when navigating to the Tokens and Airdrops home screens prior to the Owner token being minted.

**EXISTING: Community founder view of Tokens and Airdrops home screens:**
The Community founder sees a `Get started` panel which introduces them to the concept of the Owner token, explaining why minting is disabled until the Owner token is minted. It also contains a CTA to `Mint Owner token`

Figma link:
https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?type=design&node-id=34794-589770&mode=design&t=H9WXPVH15Vuk0mSS-4

<img width="1440" alt="Tokens" src="https://github.com/status-im/status-desktop/assets/110033914/77b74ec0-7ea6-4e41-bcdd-43ca78a26b3d">

<img width="1440" alt="Airdrop" src="https://github.com/status-im/status-desktop/assets/110033914/549a68aa-c091-4847-85ee-60e1b9350ecb">

**NEW: Admin view of Tokens and Airdrops home screens:**
The admin view of these screens (before the Owner token is minted) contains a modified version of the `Get started` panel. The copy is adjusted to introduce the concept of the TokenMaster token to them and to encourage them to reach out to the Community founder if they would like TokenMaster permissions.

Figma link:
https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?type=design&node-id=34794-589843&mode=design&t=H9WXPVH15Vuk0mSS-4

<img width="1440" alt="Tokens" src="https://github.com/status-im/status-desktop/assets/110033914/592606d4-e511-41d2-ac55-36f059d91410">

<img width="1440" alt="Airdrop" src="https://github.com/status-im/status-desktop/assets/110033914/98366349-229b-47a5-a5d4-74e6e284ffee">

Once the Community Owner and TokenMaster tokens have been minted, the Owner `Get started` panel is removed from the UI. However, the admin view of the `Get started` panel should persist even after the Owner token is minted given it is still relevant.

cc @noeliaSD

**Comment by noeliaSD** [comment]
> Once the Community Owner and TokenMaster tokens have been minted, the Owner Get started panel is removed from the UI. However, the admin view of the Get started panel should persist even after the Owner token is minted given it is still relevant.

@benjthayer I have my doubts here bc when the `Owner` and `TokenMaster` tokens have been minted, the welcome / initial screen is no longer shown (for `Tokens` section) and instead, we have [this](https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?type=design&node-id=34794-590512&mode=design&t=fyQZydOUKjYb8Di9-0):

![Screenshot 2023-07-12 at 14 29 27](https://github.com/status-im/status-desktop/assets/97019400/172cc91e-0dce-4bc4-b3ab-1c95c0bf23dc)

Maybe we can ONLY keep the `Get started` panel in the `airdrop` section. WDYT?

**Comment by benjthayer** [comment]
> Maybe we can ONLY keep the `Get started` panel in the `airdrop` section. WDYT?

Good point @noeliaSD ! Yes, I think that makes sense. Thanks :)

In future, the airdrop home will actually be replaced with an airdrop history (reverse chronological list of past airdrops) so when that comes into play, we can perhaps remove the panel from the Airdrop home after the owner token is minted as well.

One thing I spotted on reviewing the attached screens, for admins, the copy in the `Get started` panel for airdropping should relate to airdropping not minting. I've updated the copy:

https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?type=design&node-id=34794-589990&mode=design&t=7HSjl1pkHU0oNfhu-4
